docker-compose.yml

```diff
@@ -119,7 +119,7 @@ services:
     command: ["open", "-C", "cypress/cypress.config.js"]
     volumes:
       #mac
-      - /tmp/.X11-unix:/tmp/.X11-unix
+#      - /tmp/.X11-unix:/tmp/.X11-unix
```

Comment:
Are you 100% sure by rebuilding the container --build? I don't see how it can work if there isn't a socket.

FYI - On CI the integration tests run on a Linux VM now and I believe the Cypress default VM image is X11 not xorg (~/.Xauthority) based so this socket will be defined same as it is on Mac. Probably fine to leave it in, assuming CI passes.

Reply:
So docker volumes will only get mounted at runtime, not at build time. I've tested it during runtime at it works fine without it. It's not needed in CI as this is only for displaying the cypress UI if we want it, shouldn't need it in CI pipeline